[REF] estate_property: merge menus and update date logic

JemXiaolong opened the pull request:
- Eliminated estate_menus.xml and merged its content into
  estate_property_views.xml to simplify view management.

- Replaced datetime.today() with fields.Date.today() to
  automatically respect user's timezone.

- Removed 'models': ['estate.property'] from manifest as
  it was unnecessary.

xmglord commented on estate/__manifest__.py:
```diff
@@ -0,0 +1,23 @@
+    'security/ir.model.access.csv',
+],
+
+    'models':['estate.property'],
```
x2

xmglord commented on estate/models/estate_property.py:
```diff
@@ -0,0 +1,40 @@
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(
+        default=lambda self: datetime.today() + timedelta(days=90),
```
x2

xmglord commented on estate/views/estate_menus.xml:
```diff
@@ -0,0 +1,7 @@
+<odoo>
```
in same file as the rest of the view

xmglord reviewed (changes requested):
- Use [OCA guidelines](https://www.odoo.com/documentation/saas-18.2/contributing/development/git_guidelines.html) for commit title.
- In description you have \n instead of the break line.
- All comments and string should be in English and you should add a translation file (.po) file for the translation, example [here](https://www.youtube.com/watch?v=ZRpmM8lDn-w).
- Use separate commits for different changes, in this case the marin changes should be in another commit.

JemXiaolong committed:
[REF] estate_property: merge menus and update date logic

- Eliminated estate_menus.xml and merged its content into
  estate_property_views.xml to simplify view management.

- Replaced datetime.today() with fields.Date.today() to
  automatically respect user's timezone.

- Removed 'models': ['estate.property'] from manifest as
  it was unnecessary.

JemXiaolong committed:
[I18N] estate/i18n: add Spanish translation file es_MX.po

JemXiaolong committed:
[IMP] product_template: add test attribute and sales section

- Added test field 'property_account_jem' in product_template.py.

- Created a new section in the sales tab of product_template_views.xml
  to display price, availability, sellers, and product tags.

- Only test changes to validate UI and additional fields."

xmglord commented on estate/__manifest__.py:
```diff
@@ -0,0 +1,21 @@
+        'base'
+        ],
+
+'data': [
```
This is unindent

xmglord commented on estate/__manifest__.py:
```diff
@@ -0,0 +1,21 @@
+    'application': True,
+    'auto_install': False,
+}
+
```
3 blank lines at the end of file is not expected

xmglord commented on estate/models/estate_property.py:
```diff
@@ -0,0 +1,40 @@
+        ('offer_accepted', 'Offer Accepted'),
+        ('sold', 'Sold'),
+        ('cancelled', 'Cancelled'),
+    ], string="State", default='new', required=True, copy=False)
```
this is unindented  the ] should be at the same level as the line above

xmglord commented on estate/i18n/es_MX.po:
```diff
@@ -0,0 +1,183 @@
+#. module: estate
+#: model:ir.model.fields,field_description:estate.field_estate_property__active
+msgid "Active"
+msgstr ""
```
The translation is missing in all the file, do not add it, is not worth it but just to know the translation should be here.

xmglord commented on marin/models/product_template.py:
```diff
@@ -36,6 +36,15 @@ class ProductTemplate(models.Model):
         "Leave empty to use the account from the product category.",
     )
+
+    #inicio creación de atributos
```
This is not necessary also the comments should start with a blank space `# comment`

xmglord reviewed (changes requested):
First commit:
- The module is estate not estate_property
- The description should be what you do in the commit, not the changes requested

Second commit:
- Don't specify the file, just the module estate

Third commit:
- Module is marin not product_template
- Unnecessary quote (") at the end of description

PR title:
- Use OCA guidelines
